browser/main/lib/ConfigManager.js

```diff
@@ -21,7 +21,8 @@ export const DEFAULT_CONFIG = {
   amaEnabled: true,
   hotkey: {
     toggleFinder: OSX ? 'Cmd + Alt + S' : 'Super + Alt + S',
-    toggleMain: OSX ? 'Cmd + Alt + L' : 'Super + Alt + E'
+    toggleMain: OSX ? 'Cmd + Alt + L' : 'Super + Alt + E',
+    timestamp: OSX ? 'Cmd + Alt + T' : 'Super + Alt + T'
```

Comment:
I don't think `timestamp` should be located in `hotkey` because `hotkey` was implemented based on the idea that it operates when a user doesn't focus on the application.